src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpgradeForm.tsx

```diff
@@ -116,16 +121,16 @@ function UpgradeForm({ selectedPlan, setSelectedPlan }: UpgradeFormProps) {
         register={register}
         errors={errors}
       />
+      <UpdateBlurb
+        currentPlan={currentPlan}
+        selectedPlan={selectedPlan}
+        newPlanName={newPlan}
+        seats={Number(seats)}
+        nextBillingDate={getNextBillingDate(accountDetails)!}
```

Comment:
when the user hits manage plan page they'll always have a plan, so I'm not too concerned about the assertion here

Reply:
yea makes sense.